subprojects/docs/src/samples/groovy/incrementalCompilation/kotlin/build.gradle.kts

```
@@ -0,0 +1,20 @@
+subprojects {
+    apply(plugin = "groovy")
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        "implementation"(localGroovy())
+    }
+// tag::enable-groovy-incremental[]
+    tasks.withType<GroovyCompile> {
```

```suggestion
    tasks.withType<GroovyCompile>().configureEach {
```

Is this eager or lazy?